app/models/person.rb

```diff
@@ -37,4 +37,17 @@ def roles_inclusion
       errors.add(:roles,"'#{invalid_role}' is a invalid role.")
     end
   end
+  
+  def find_current_institutions
+	current_affiliations = self.affiliations.where("year_to <= #{Date.today.year} OR year_to IS NULL").all
```

Comment:
Raczej `year_to >= Date.today.year` (to jest ostatni rok pracy w danej instytucji, zatem musi być >= od bieżącego roku). Musi być też `year_from` (może być w osobnym `where` ponieważ łączone są za pomocą `AND`).